Angular/src/app/helpers/quiz-recipe-helper.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class QuizRecipeHelperService {
+
+  formatAiReponseQuizString(quizString: any) {
+    const quizData: any = {
```

Comment:
Also no `any`

Reply:
that was helper only, but no worries i fixed.